src/GRA.Controllers/EventsController.cs

```diff
@@ -107,7 +129,19 @@ public class EventsController : Base.UserController
                     filter.SortBy = SortEventsBy.Distance;
                 }
             }
-
+            if (AuthUser.Identity.IsAuthenticated)
+            {
+                filter.Favorites = Favorites;
+                if (string.IsNullOrWhiteSpace(Visited)
```

Comment:
It seems like all that needs to happen here is set it to true if Visited is equal to VisitedYes. If it's set to the default initialization value it will be false already. Unless it's nullable?

Reply:
Part of @iafb's changes

Reply:
IsAttended is a trinary with null being the default option, the value will not be set unless Visited is empty or matches one of the string constants.